modules/lazyload/assets/scss/frontend.scss

```diff
@@ -1,3 +1,8 @@
-body.e-lazyload .lazyloaded, body.e-lazyload .lazyloaded .elementor-background-overlay {
+.lazyloaded:not( .elementor-motion-effects-element-type-background ),
```

Comment:
Please add comments that explain the selectors' targets.

Reply:
done